addons/sourcemod/scripting/CallAdmin.sp

```diff
@@ -368,28 +292,197 @@ stock void SendWebHook(int userid, char sMessage[4096], char sWebhookURL[WEBHOOK
 		}
 	}
 
+	/* Webhook UserName */
 	char sName[128];
 	g_cvUsername.GetString(sName, sizeof(sName));
 
+	/* Webhook Avatar */
 	char sAvatar[256];
 	g_cvAvatar.GetString(sAvatar, sizeof(sAvatar));
 
+	/* Map Name */
+	char sMapName[PLATFORM_MAX_PATH], sMapNameLower[PLATFORM_MAX_PATH];
+	GetCurrentMap(sMapName, sizeof(sMapName));
+	GetCurrentMap(sMapNameLower, sizeof(sMapNameLower));
+
+	/* Map Time Left */
+	int timeleft;
+	char sTimeLeft[32];
+	if (GetMapTimeLeft(timeleft))
+	{
+		if (timeleft > 0)
+			FormatEx(sTimeLeft, sizeof(sTimeLeft), "%i:%02i", timeleft / 60, timeleft % 60);
+		else if (timeleft <= 0)
+			FormatEx(sTimeLeft, sizeof(sTimeLeft), "Last Round");
+	}
+	else FormatEx(sTimeLeft, sizeof(sTimeLeft), "N/A");
+
+	/* Team score */
+	char sTeamScore[64];
+	FormatEx(sTeamScore, sizeof(sTeamScore), "%s %d - %d %s", 
+		g_Plugin_ZR ? "Humans" : "CTs", CS_GetTeamScore(CS_TEAM_CT),
```

Comment:
instead of each `g_Plugin_ZR ?`, you could save the result in a variable like this:
```
sTeamCTName = g_Plugin_ZR ? "Humans" : "CTs"
sTeamTName = g_Plugin_ZR ? "Zombies" : "Ts"
```
and then use it when needed